src/ui/src/index.html

```diff
@@ -173,17 +173,23 @@
                   </a>
                 </li>
                 <li ng-show="hasPermission(user, 'bg-user-read')">
-                  <a ui-sref="user_admin" ui-sref-active="active">
+                  <a ui-sref="base.user_admin" ui-sref-active="active">
                     <i class="fa fa-user"></i>
                     <span>User Management</span>
                   </a>
                 </li>
                 <li ng-show="hasPermission(user, 'bg-role-read')">
-                  <a ui-sref="role_admin" ui-sref-active="active">
+                  <a ui-sref="base.role_admin" ui-sref-active="active">
```

Comment:
We didn't have the prefix `base` here before. Why add it now? Was this an issue with how the UI worked previously or something that adding the `namespaces` to the UI introduced?